tests/test_hear_cleanup.py

```
@@ -166,6 +168,67 @@
+
+	def test_should_invoke_kill_func_when_handle_error_called(self) -> None:
+		"""Test that kill_func calls shutdown on the server instance.
+
+		Verifies that the server properly handles requests without
+		the STOP command and calls the kill_func to free up server resources.
+		"""
+		theResult = False
+		fail_fixture = "Mock(STOP) --> Handler-HEAR --X shutdown"
+		_fixture_port_num = self._the_test_port
+		try:
+			self.assertIsNotNone(_fixture_port_num)
```

Ruff (0.11.9)

Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)